rc-events-service/src/main/scala/com/epam/random_coffee/events/services/impl/EventServiceImpl.scala

```diff
@@ -0,0 +1,42 @@
+package com.epam.random_coffee.events.services.impl
+
+import com.epam.random_coffee.events.model.{ Event, EventId }
+import com.epam.random_coffee.events.repo.EventRepository
+import com.epam.random_coffee.events.services.EventService
+
+import java.util.UUID
+import scala.concurrent.{ ExecutionContext, Future }
+
+class EventServiceImpl(repo: EventRepository)(implicit ec: ExecutionContext) extends EventService with Doobie_support {
```

Comment:
Why does your service extends `DoobieSupport`? Why does it need it?

Reply:
Not needed and deleted